fee-calculator/src/lib.rs

```diff
@@ -237,9 +233,6 @@ mod tests {
         loop {
             let last_lamports_per_signature = f.lamports_per_signature;
 
-            f = FeeRateGovernor::new_derived(&f, u64::MAX);
```

Comment:
Did you mean to remove this line?

Reply:
I did not, corrected in 8f9d1a8f14a7fe2c9cfc9baf4c181784dcd22db0